<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frappy-gui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1228</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Node</string>
    </property>
    <widget class="QMenu" name="menuRecent_SECNodes">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Recent SECNodes</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionClear"/>
    </widget>
    <addaction name="actionAdd_SEC_node"/>
    <addaction name="menuRecent_SECNodes"/>
    <addaction name="actionReconnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionShow_Logs"/>
    <addaction name="actionDetailed_View"/>
    <addaction name="actionHighlightAnimation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_SEC_node"/>
  </widget>
  <action name="actionAdd_SEC_node">
   <property name="icon">
    <iconset resource="../../../resources/frappy-gui.qrc">
     <normaloff>:/icons/connect</normaloff>:/icons/connect</iconset>
   </property>
   <property name="text">
    <string>Open SECNode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionShow_Logs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Log Window</string>
   </property>
  </action>
  <action name="actionAdvanced_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Advanced Mode</string>
   </property>
  </action>
  <action name="actionDetailed_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Detailed View</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../../../resources/frappy-gui.qrc">
     <normaloff>:/icons/trash</normaloff>:/icons/trash</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actiondjhfs">
   <property name="text">
    <string>djhfs</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="text">
    <string>Reconnect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionHighlightAnimation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highlight name upon selection</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../resources/frappy-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
